README.md

```diff
@@ -26,7 +26,7 @@ Each MinIO Tenant represents an independent MinIO Object Store within
 the Kubernetes cluster. The following diagram describes the architecture of a
 MinIO Tenant deployed into Kubernetes:
 
-IMAGE
+<img src=docs/images/architecture.png width="600px"/>
```

Comment:
```suggestion
<img src=docs/images/architecture.png width="600px"/>
```

You should use markdown syntax here @ravindk89 using HTML is not advised generally

Reply:
Done